### PR TITLE
libvirt: Update to 4.1.0

### PR DESCRIPTION
**Pull request by raimue**
#### Description

Update libvirt and py-libvirt to 4.1.0.

Closes: https://trac.macports.org/ticket/52968

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 10.12.6 16G1212
Xcode 9.2 9C40b

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: sysutils/libvirt/Portfile [hunk 1]
```diff
@@ -17,56 +16,86 @@ long_description    A toolkit to interact with the virtualization \
 
 homepage            https://libvirt.org
 master_sites        ${homepage}/sources/
+use_xz yes
```
Comment (ryandesign) [inline sysutils/libvirt/Portfile]: Can we move this `yes` over to match the indentation of the other lines?

File: sysutils/libvirt/Portfile [hunk 2]
```diff
@@ -75,11 +104,35 @@ variant avahi description {Use Avahi to advertise remote daemon} {
...
+variant fuse description {FUSE support} {
+    depends_lib-append      port:osxfuse
+    configure.args-replace  --without-fuse --with-fuse
+}
+
+variant openssl description {Use OpenSSL instead of GnuTLS} {
+    depends_lib-replace     port:gnutls path:lib/libssl.dylib:openssl
+    configure.args-replace  --with-gnutls --without-gnutls
+    configure.args-replace  --without-openssl --with-openssl
+}
```
Comment (ryandesign) [inline sysutils/libvirt/Portfile]: I would prefer to also add a gnutls variant, and for the openssl and gnutls variants to conflict.

File: sysutils/libvirt/Portfile (hunk 1 above)
Comment (raimue) [reply sysutils/libvirt/Portfile]: Done.

**Comment by pmetzger** [comment]
Unless there's reason not to, I'll merge this later today.

**Comment by pmetzger** [comment]
@raimue Just being explicit, you're not still waiting for more feedback on this one are you?

**Comment by raimue** [comment]
@ryandesign already approved the changes to libvirt. The update to py-libvirt falls under the openmaintainer policy. So, no need to wait, let me go ahead and merge it. Thanks for asking.

**Pull request merged**